src/fmu/dataio/_utils.py

```diff
@@ -82,6 +85,15 @@ def export_metadata_file(
             "Export of metadata was requested, but no metadata are present."
         )
 
+    try:
+        meta.Root.model_validate(metadata)
+    except ValidationError as e:
```

Comment:
Would be epic if we have sentry or similar tools that could pick up these errors and logged them somehow.

Reply:
Log warning needs to be conditional, only log during FMU-run.